packages/now-static-build/test/test.js

```diff
@@ -7,50 +7,33 @@ const {
 } = require('../../../test/lib/deployment/test-deployment.js');
 
 jest.setTimeout(12 * 60 * 1000);
-let buildUtilsUrl;
 let builderUrl;
 
 beforeAll(async () => {
-  if (!buildUtilsUrl) {
-    const buildUtilsPath = path.resolve(__dirname, '..');
-    buildUtilsUrl = await packAndDeploy(buildUtilsPath);
-    console.log('buildUtilsUrl', buildUtilsUrl);
```

Comment:
Let's add `buildUtilsUrl` back like we discussed (although you can fix the path)